### ISSUE TITLE
File permission in gem version 1.8.6 broken

### ISSUE THREAD

**Issue by erdgeist** [bulletproofnetworks/roust#25]
When fetching the gem https://rubygems.org/gems/roust/versions/1.8.6 at https://rubygems.org/downloads/roust-1.8.6.gem, I found some files having user-only access permission (in the internal tar archive, already):

```
 24 -rw-------    1 root         wheel               12233 14 Mai  2015 ./lib/roust/ticket.rb
  8 -rw-r--r--    1 root         wheel                2449 14 Mai  2015 ./lib/roust/user.rb
  8 -rw-------    1 root         wheel                  36 14 Mai  2015 ./lib/roust/version.rb
 16 -rwx------    1 root         wheel                4147 14 Mai  2015 ./lib/roust.rb
  8 -rw-r--r--    1 root         wheel                  591 14 Mai  2015 ./LICENSE
```

If installed by bundle as user root, my rails app – running as a non-privileged user – can not access those files and the require 'roust' will fail with:

`LoadError (cannot load such file -- roust):`

**Comment by nerdyness** [comment]
Solved in #26 

Thanks again, @erdgeist :+1:

**Comment by erdgeist** [comment]
Looks like it's working. Thank you.
